commcare_connect/opportunity/tasks.py

```
@@ -494,18 +510,17 @@ def bulk_update_payments_task(self, opportunity_id: int, file_path: str, file_fo
                 rows = list(dataset)
 
             status = bulk_update_payments(opportunity_id, headers, rows)
-            messages = [f"Payment status updated successfully for {len(status)} users."]
-            if status.missing_users:
-                messages.append(status.get_missing_message())
-
+            message, is_error = get_payment_import_result_message(status)
         except ImportException as e:
-            messages = [f"Payment Import failed: {e}"] + getattr(e, "invalid_rows", [])
+            message = "<br>".join([f"Payment Import failed: {e}"] + getattr(e, "invalid_rows", []))
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)